styles/mixins/menu-slide.css

```diff
@@ -102,4 +102,38 @@ title: Mobile Menu
       }
     }
   }
+  &.mobile-only {
```

Comment:
Its now using a newer custom media query ensures that using `tablet-down` will force it regardless of media query approach.